### PR TITLE
Add SDK class with single instance

### PR DESCRIPTION
**Pull request by wachunei**

### CODE REVIEW

**Review by VikChawla — APPROVED** [review]
lgtm

File: app/components/UI/FiatOnRampAggregator/sdk/SDK.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import { OnRampSdk, IOnRampSdk } from '@chingiz-mardanov/on-ramp-sdk';
+
+class SDK {
```
Comment (alaahd) [inline app/components/UI/FiatOnRampAggregator/sdk/SDK.ts:3]: I suggest we rename the class to `Instance` and the file name as well.

Comment (wachunei) [reply app/components/UI/FiatOnRampAggregator/sdk/SDK.ts:3]: `SDKInstance` ?

**Comment by wachunei** [comment]
Closing this until we solve the split of the SDK into orders and buy flow.

**Pull request closed without merging**